src/main/java/com/fasterxml/jackson/core/JsonGenerator.java

```diff
@@ -2977,4 +3008,18 @@ protected void _writeSimpleObject(Object value) throws IOException
         throw new IllegalStateException("No ObjectCodec defined for the generator, can only serialize simple wrapper types (type passed "
                 +value.getClass().getName()+")");
     }
+
+    /**
+     * Returns a new {@code int[]} of escape table with additional forward-slash escaping configured
+     * as configured by {@link JsonWriteFeature#ESCAPE_FORWARD_SLASHES}.
+     *
+     * @since 2.17
+     */
+    protected int[] escapeForwardSlash(int[] escapes) {
```

Comment:
Should not be added here since this is not JSON-specific type despite name. So rather add in `JsonGeneratorImpl`, shared base class for concrete ones